client/posts/embed.ts

```diff
@@ -35,7 +43,7 @@ const formatters: { [key: number]: (s: string) => string } = {}
 // Map of providers to information fetcher functions
 const fetchers: { [key: number]: (el: Element) => Promise<void> } = {}
 
-for (let p of ["Youtube", "SoundCloud", "Vimeo"]) {
+for (let p of ["Youtube", "SoundCloud", "Vimeo", "Hooktube"]) {
```

Comment:
This still calls into the noembed.com API (`formatNoEmbed()` and `fetchNoEmbed()`). You will need figure out how to handle the hooktube API and write separate functions for that.